crates/containerd-shim-wasm/src/sandbox/shim/local.rs

```diff
@@ -34,9 +34,6 @@ use crate::sandbox::shim::instance_data::InstanceData;
 use crate::sandbox::{Error, Result, oci};
 use crate::sys::metrics::get_metrics;
 
-#[cfg(test)]
-mod tests;
-
```

Comment:
This enables the tests in `src/sandbox/shim/local/tests.rs`.
Could you move the new tests into that file instead?